README.md

```diff
@@ -111,6 +112,60 @@ module "rds_cluster_aurora_mysql" {
 }
 ```
 
+With [enhanced monitoring](examples/enhanced_monitoring)
+
+```hcl
+# create IAM role for monitoring
+resource "aws_iam_role" "iam_role" {
+  name               = "rds-${var.cluster-name}"
+  assume_role_policy = "${data.aws_iam_policy_document.policy_document.json}"
+}
+
+# attach amazon's managed policy for RDS to write logs
+resource "aws_iam_role_policy_attachment" "policy_attachment" {
+  role       = "${aws_iam_role.iam_role.name}"
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole"
+}
+
+# allow rds to assume this role
+data "aws_iam_policy_document" "policy_document" {
```

Comment:
`policy_document` does not contribute any new information. The data source is already an `aws_iam_policy_document`. Lets rename to `enhanced_monitoring`.

Reply:
I'm fairly new to terraform, I have just been giving names that correlate to the resource type.  I give meaningful names once I have conflicts between multiple resources.
I'll change this name in the example.

Reply:
What we've done in that case, is use the keyword `default` if there's only one of something. That's also ok. =)